tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CONST.py

```diff
@@ -31,23 +37,28 @@
 STR_ERR_MSG = "Error message is: "
 STR_ADD_RECEPTORS_SUCCESS = "Resources are assigned successfully on CspSubarray."
 STR_REMOVE_ALL_RECEPTORS_SUCCESS = "All resources assigned to CSP Subarray are removed successfully."
-STR_CONFIGURESCAN_SUCCESS = "ConfigureScan invoked successfully on CspSubarray."
+STR_CONFIGURESCAN_SUCCESS = "ConfigureScan command invoked successfully on CspSubarray from " \
```

Comment:
I am not sure but this line may give error... Will it work if we don't concatenate (add "+") the strings?

Reply:
It is working.